indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -55,7 +55,7 @@
             'button[data-href][data-method]', 'input:button[data-href][data-method]', 'a[data-href][data-method]',
             'button[data-href][data-ajax-dialog]', 'input:button[data-href][data-ajax-dialog]', 'a[data-href][data-ajax-dialog]'
         ];
-        $('body').on('click', selectors.join(', '), function(e) {
+        $('body').on('click', selectors.join(':not(.disabled), ') + ':not(.disabled)', function(e) {
```

Comment:
`_.map(selectors, function(x) { return x + ':not('.disabled')' }).join(', ')` might be clearer